scripts/regression/test_runner.py

```diff
@@ -133,32 +143,62 @@ class BenchmarkResult:
         ):
             regression_list.append(BenchmarkResult(benchmark, old_res, new_res))
         else:
-            other_results.append(BenchmarkResult(benchmark, old_res, new_res))
+            improved_by = int((new_res - old_res) * 100.0 / new_res)
```

Comment:
If this is an improvement, wouldn't `new_res - old_res` always be negative? If a query goes from 10s to 8s, we would get 8 - 10 = -2.